### ISSUE TITLE
Set Ruby 2.5 as minimum ruby?

### ISSUE THREAD

**Issue by hainesr** [rubyzip/rubyzip#484]
With the move to version 3.0 we have the opportunity to bump the minimum version of ruby that we support. Should we bump up to 2.5? We currently have 2.4 as the minimum, but this went EOL back in March 2020. I'm happy to support older rubies, but maybe one version EOL is enough, and 2.5 is EOL now.

What do people think? Especially keen to hear thoughts from @jdleesmiller and @simonoff.

**Comment by hainesr** [comment]
If we don't set 2.5 as minimum supported ruby now, then we'd have to wait for rubyzip 4.0, and that could be a way off. 2.4 has been EOL for nearly 2 years now.

**Comment by petergoldstein** [comment]
Frankly I'd suggest 2.6 as a minimum, given that it's going to be EOL in ~2 months (https://endoflife.date/ruby)

**Comment by hainesr** [comment]
Yes, I was certainly tempted! I'll go with 2.5 for now, and if it takes me much longer to get version 3 out it might creep up to 2.6 by then 😆
